libr/anal/esil.c

```diff
@@ -1904,6 +1793,8 @@ static int esil_mem_modeq_n(RAnalEsil *esil, int bits) {
 			src1 = r_anal_esil_pop (esil);
 			if (src1 && r_anal_esil_get_parm (esil, src1, &d)) {
 				r_anal_esil_pushnum (esil, d % s);
+				d = d % s;
```

Comment:
its just an if, you can change that now

> On 23 May 2016, at 16:26, Sushant Dinesh <notifications@github.com> wrote:
> 
> In libr/anal/esil.c <https://github.com/radare/radare2/pull/4951#discussion_r64231502>:
> 
> > @@ -1904,6 +1793,8 @@ static int esil_mem_modeq_n(RAnalEsil *esil, int bits) {
> >  			src1 = r_anal_esil_pop (esil);
> >  			if (src1 && r_anal_esil_get_parm (esil, src1, &d)) {
> >  				r_anal_esil_pushnum (esil, d % s);
> > +				d = d % s;
> I didn't make any changes here. I merely reverted the commit. I'll look into it though. Maybe I'll make those into separate commits.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub <https://github.com/radare/radare2/pull/4951/files/46fd36b8d9084d1cb2d6d7bd6dc8faea31735481#r64231502>